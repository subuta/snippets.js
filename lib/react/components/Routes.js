import { build, format, snippets as s } from 'bld.js'
import _ from 'lodash'

export default () => {
  const imports = s.import([
    ['react', 'React'],
    ['_', 'lodash'],
    ['react-router-redux', null, [
      'ConnectedRouter'
    ]],
    ['src/utils/history', 'history']
  ])

  return build`
    ${imports}
    
    /* mat Custom imports [start] */
    /* mat Custom imports [end] */
    
    let routes = null
    
    /* mat Custom routes [start] */
    /* mat Custom routes [end] */
    
    export default routes

  `
}