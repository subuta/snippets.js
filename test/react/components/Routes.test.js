/* global expect, describe, it */

import { build, format, snippets as s } from 'bld.js'

import Routes from 'lib/react/components/Routes'

describe('react/components/Routes', () => {
  it('should create react Routes', () => {
    expect(format(Routes())).toMatchSnapshot()
  })
})
