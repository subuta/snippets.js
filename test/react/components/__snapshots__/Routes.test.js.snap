// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`react/components/Routes should create react Routes 1`] = `
"import React from 'react'
import _ from 'lodash'
import {ConnectedRouter} from 'react-router-redux'
import history from 'src/utils/history'

/* mat Custom imports [start] */
/* mat Custom imports [end] */

let routes = null

/* mat Custom routes [start] */
/* mat Custom routes [end] */

export default routes
"
`;
